[FIX] helpdesk_mgmt: automatic stage

cristina-domatix opened the pull request:
The date in which the ticket had to change its stage was been computed but, apparently,
not stored properly, so the ticket moved every time the cron was executed.
We fixed this by setting a computed field for the date the ticket had to change its stage.

cristina-domatix committed:
[FIX] helpdesk_mgmt: automatic stage
The date in which the ticket had to change its stage was been computed but, apparently,
not stored properly, so the ticket moved every time the cron was executed.
We fixed this by setting a computed field for the date the ticket had to change its stage.

cristina-domatix committed:
[FIX] helpdesk_mgmt: travis tests

catalin-domatix commented on helpdesk_mgmt/models/helpdesk_ticket.py:
```diff
@@ -149,6 +149,33 @@ def _read_group_stage_ids(self, stages, domain, order):
+        string="Next stage date", compute="_compute_next_stage"
+    )
+
+    @api.depends("stage_id")
```
@cristina-domatix i think it can by done while depending only on `stage_id.auto_last_update` and `stage_id.auto_next_number` and not the full object

catalin-domatix replied:
@cristina-domatix i think it can be done while depending [only on the needed attributes](https://github.com/odoo/odoo/blob/13.0/odoo/api.py#L184) (`auto_next_number`, `auto_next_type`, `auto_next_stage_id`) and not the full `stage_id` object as to avoid unnecesary calculations

cristina-domatix committed:
[FIX] helpdesk_mgmt: dependency of the compute method
The computed method for the stage date only depends on the required attributes, not the whole
object.
